src/components/Editor/Editor.less

```diff
@@ -131,6 +131,18 @@
         text-align: right !important;
       }
     }
+
+    &__cancel {
+      margin-bottom: 0 !important;
+      padding-right: 15px;
```

Comment:
It feel there is too much space between the 2 buttons. Can you use `10px` instead of `15px`?